src/flatbuffers/metric_common.fbs

```diff
@@ -0,0 +1,57 @@
+namespace circonus;
+
+table IntValue {
+      value:int (id: 0);
+}
+
+table UintValue {
+      value:uint (id: 0);
+}
+
+table LongValue {
+      value:long (id: 0);
+}
+
+table UlongValue {
+      value:ulong (id: 0);
+}
+
+table DoubleValue {
+      value:double (id: 0);
+}
+
+table StringValue {
+      value:string (id: 0);
+}
+
+// intentionally empty.. Just a Union placeholder
+table AbsentNumericValue {
+}
+
+// intentionally empty.. Just a Union placeholder
+table AbsentStringValue {
+}
+
+// intentionally empty.. Just a Union placeholder
+table AbsentHistogramValue {
+}
+
+table HistogramBucket {
+       val: byte;
+       exp: byte;
+       count: ulong;
+}
+
+table Histogram {
+       buckets: [HistogramBucket] (id: 0);
+}
+
+union MetricValueUnion { IntValue, UintValue, LongValue, UlongValue, DoubleValue, StringValue, Histogram, AbsentNumericValue, AbsentStringValue, AbsentHistogramValue }
+
+table MetricValue {
+      name: string (id: 0);
+      timestamp: ulong (id: 1);
```

Comment:
not that familiar with flatbuffers, but it appears that `name` and `timestamp` are repeated in `table MetricValue` (in this file) and `table Metric` (in metric.fbs). I suspect we should delete the fields from `MetricValue`?